contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1264,6 +1342,50 @@ export const AssessmentItem = new Resource({
       ...id,
     });
   },
+
+  delete(id) {
+    return this.transaction('rw', TABLE_NAMES.CONTENTNODE, () => {
+      const nodeId = id[0];
+      return this.table.delete(id[1]).then(data => {
```

Comment:
Sorry, I missed this in my first pass through - this needs to be the whole id. The unique id for each assessmentitem in the indexddb is composed of `[<contentnode>, <assessment_id>]`.

Reply:
(have updated)